### PR TITLE
add-mines-test PR suggestions

### PR DESCRIPTION
**Pull request by rkamysz**
Since there are problems with running the tests after seeding, I have allowed myself to create this PR with suggestions to make it easier to spot the differences.

1. I changed the contents of the `run-api-tests.sh` file. The process is a bit more complex now, after the build we run the containers in the background, then we get the test-runner id and log the results, finally cleaning and returning the code. The code itself is more of a case if this process were to be part of the CI.
2. I replaced `Dockerfile.mongo.seed` with stages in the already existing one api tests`Dockerfile.api.tests` - less files
3. I moved seed file to `tests/api/fixtures`

I have the biggest problem with seeding for testing ...
We could use a stub, but then we don't test the connection to the database (like real case) or maybe timeouts etc.
For similar tests, I used a test database instance available on staging (internally), we do not have it at the moment, so we seed instead of passing a specific url in the config.

What do you think about this? Keep in mind that we do not have any e2e/integration tests between services and other systems, so this is temporarily our only form of checking whether our changes have had a negative effect on the highest level of alienworlds-api. Compare it to manual tests performed with, for example, a Postman or a simple curl.

### CODE REVIEW

File: scripts/run-api-tests.sh [hunk 1]
```diff
@@ -1,4 +1,16 @@
...
+TEST_EXIT_CODE=`docker wait $TESTS_RUNNER_CONTAINER_ID`
+
+# Log tests results
+docker logs $TESTS_RUNNER_CONTAINER_ID
+
+# Clean
+docker-compose -p tests kill
+docker-compose -p tests rm -f
+
+exit $TEST_EXIT_CODE
```
Comment (ktamas77) [inline scripts/run-api-tests.sh:16]:
```suggestion
exit $TEST_EXIT_CODE

```

**Comment by ktamas77** [comment]
Great job adding multi-stage build support!

**Comment by ktamas77** [comment]
A stub is 100% acceptable, in fact the desired solution to run unit tests. For integration tests we can initialize a mock database later (in the local docker network) which could be tested both locally, on CI & in the future on staging as well.

**Review by montera82 — APPROVED** [review]

**Comment by rkamysz** [comment]
I close this PR as we already have necessary changes in another PR and on develop

**Pull request closed without merging**